chapters/planning-data-work.tex

```diff
@@ -1,147 +1,457 @@
-
-% When using a computer for research, you should keep in mind a structure of work known as \textbf{scientific computing}.\cite{wilson2014best,wilson2017good} \index{scientific computing} Scientific computing is a set of practices developed to help you ensure that the computer is being used to improve your efficiency, so that you can focus on the real-world problems instead of technical ones.\sidenote{ \url{https://www.dropbox.com/s/wqefknwfb91kop8/Coding_For_Econs_20190221.pdf?raw=1}} This means getting to know your computer a little better than most people do, and thinking critically about tasks like file structures, code and \textbf{process reusability},\sidenote{ \url{http://blogs.worldbank.org/opendata/making-analytics-reusable}} and software choice. Most importantly, it means detecting early warning signs of \textbf{process bloat}. As a general rule, if the work required to maintain a process grows as fast (or faster) than the number of objects controlled by that process, you need to stop work immediately and rethink processes. You should work to design processes that are close to infinitely scalable by the number of objects being handled -- whether they be field samples, data files, surveys, or other real or digital objects.  The first thing you need to figure to use your computer efficiently is where you are on your file system. \marginnote{You should \textit{always} use forward slashes (\texttt{/}) in file paths. Backslashes will break folder paths in many systems.}
-
-Find your \textbf{home folder}. On MacOS, this will be a folder with your username.
-On Windows, this will be something like ``This PC''. (It is never your desktop.)
-Nearly everything we talk about will assume you are starting from here.
+  \index{password protection}
+All machines should have \textbf{hard disk encryption} enabled.
+  \index{encryption}
+Disk encryption is built-in on most modern operating systems;
+the service is currently called BitLocker on Windows or FileVault on MacOS.
```

Comment:
This is far from true. This feature is only built in in Windows Business. In the Windows Home edition this does not exist. https://support.microsoft.com/en-us/help/4028713/windows-10-turn-on-device-encryption

Reply:
what about https://www.howtogeek.com/173592/windows-8.1-will-start-encrypting-hard-drives-by-default-everything-you-need-to-know/? I'll hold on this. But to me it seems this kind of protection should be absolutely mandatory.... ugh although I realize most Android phones probably are not secure either, they should have less disk data right?

Reply:
while I agree with you in principle, I am uncomfortable to recommend a tool that is not built in to the OS that if you screw up makes you loose all your data. Should we discuss what to recommend when we meet on Tuesday?

Reply:
https://github.com/worldbank/d4di/issues/328